Add GitHub edit link to bottom of content pages

bdemers opened the pull request:
This change moves the 'edit link' from the PageTitle component to the Layout, as PageTitle is NOT use for every page

**NOTE:**
* Edit link is at the bottom of page
* It is only shown when screen size is bigger than a tablet 
* css is minimal, someone may want to improve it before merging

Example:
The browser URL: 
```
/docs/guides/sign-users-out/android/before-you-begin/
```

shows a link to 
```
https://github.com/okta/okta-developer-docs/edit/master/packages/@okta/vuepress-site/docs/guides/sign-users-out/android/before-you-begin//index.md
```

NFDEPLOY

bdemers commented:
cc: @afitnerd

jakubvul-okta commented:
@bdemers could we separate it more from the main text, and make it at least bigger/more prominent?

![Screen Shot 2020-10-13 at 3 44 24 PM](https://user-images.githubusercontent.com/26014374/95908386-06b32e80-0d6b-11eb-86c5-0bf45dd6c82c.png)

alexdahl-okta commented:
👏 I support being agile here!

+1 to more top and bottom margin.
Perhaps `margin: 4rem 0 2rem 0;` or so?

bdemers commented:
Great suggestions!

More space (that looks much better, thanks @alexdahl-okta !)
![image](https://user-images.githubusercontent.com/99954/96002757-7971e780-0e07-11eb-917c-cd37f8334ec1.png)

And an increased text size (120%)
![image](https://user-images.githubusercontent.com/99954/96002892-a4f4d200-0e07-11eb-9cc9-ee3200bea7e2.png)

alexdahl-okta reviewed (commented):
Style looks fine to start with!

Should I expect it to work locally? Many guides do not.
For example: http://localhost:8080/docs/guides/sign-into-web-app/aspnet/configure-packages/ links to https://github.com/okta/okta-developer-docs/edit/master/packages/@okta/vuepress-site/docs/guides/sign-into-web-app/aspnet/configure-packages//index.md, which 404s.

bdemers commented:
Good catch @alexdahl-okta!  It took a minute to figure out why that wasn't working, it had to do with the language selector, the page loads by default without the lang value in the URL, but that doesn't work in all cases, (and when after you pick a language, doh)

fixed in: 7dae26e5a40df27f1c116d6aba5ee99da22682e2

(and rebased on master)

albertchen-okta reviewed (approved):
🚢
